projects/orangecrab/build.cfg

```diff
@@ -0,0 +1,38 @@
+###############################################################################
+# Copyright   : Copyright © 2024 QBayLogic B.V.
+# License     : MIT
+# Maintainer  : QBayLogic B.V.
+# Stability   : experimental
+# Portability : POSIX
+#
+# Required tooling for targeting the OrangeCrab r0.2.1 in combination with
+# the Pmod/Debug Expander board. Don't modify this file directly. Instead,
+# copy it to `build.cfg.local` and introduce any required changes there.
+###############################################################################
+
+CFG_PATH:=$(dir $(abspath $(lastword $(MAKEFILE_LIST))))
+
+###############################################################################
+
+# https://cabal.readthedocs.io/en/stable
+CABAL=cabal
+
+# https://github.com/clash-lang/clash-compiler
+CLASH=clash
```

Comment:
This var is unused.

I just noticed this; and I'm afraid I'll forget if I don't note it now. This is not yet the remainder of my review :-).